README.md

```diff
@@ -117,7 +117,7 @@ Change the main branch to the right one, execute the following command:
 Example commands:
 
 ```sh
-  cd my-folder
+  cd my-flder
   git clone git@github.com:myaccount/my-project.git
```

Comment:
- [ ] In your data.sql file, all code after line 24 is not necessary for this milestone and should not be present in your code base. Please remove all of it.